test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -295,6 +295,8 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
         'xpack.observability.unsafe.thresholdRule.enabled (boolean)',
         'xpack.observability_onboarding.ui.enabled (boolean)',
         'xpack.observabilityLogExplorer.navigation.showAppLink (any)', // conditional, is actually a boolean
+        'xpack.grokdebugger.enabled (any)', // conditional, is actually a boolean
```

Comment:
I'm not sure if these are needed since I removed exposing enabled to the browser. I'll see on the next CI run